gf-guide/metrics.md

```diff
@@ -50,13 +60,15 @@ GF doesn't want this because they all do it differently.
 
 *For brevity, we will refer to the 3 sets of metrics as* `Typo`*,* `Hhea`*,* `Win`*.*
 
-**3. Vertical metrics must be consistent across a family.**
+##### 5. [Use_Typo_Metrics](https://www.microsoft.com/typography/otspec/os2.htm#fss)** **must be enabled
 
-Each font in a family must share the same vertical metric values.
+This will force MS Applications to use the `Typo` values instead of the `Win` values for line spacing. By doing this, we can freely set the `Win` values to avoid clipping and control the line height with the `Typo` values. It has the added benefit of future line height compatibility. When a new script is added, we simply change the `Win` values to the new `yMin` and `yMax`, without needing to worry if the line height have changed. Note that the `Use_Typo_Metric` flag is also called `fsSelection bit 7 `(related to how it is set in the OS/2 table).
```

Comment:
Change "MS Applications" to "Microsoft applications"?

Similar to https://github.com/googlefonts/googlefonts.github.io/issues/93